### ISSUE TITLE
Close App in the first use library

### ISSUE THREAD

**Issue by AlexPGA** [coomar2841/image-chooser-library#159]
Hi , excuse me for not answering your email , I have not had time .

I have a new problem using your library. The first time I use methods to make a photo or select from the gallery , the application is closed but still alive in the background. This only happens the first time the methods used . From the second time everything works right always . Any ideas? Thanks for your reply. A greeting

**Comment by AlexPGA** [comment]
when take photo or select from gallery, the app not execute the method onImageChoosen

**Comment by coomar2841** [comment]
I am not sure if I understand your question correctly. By closing, do you mean a crash?

**Comment by AlexPGA** [comment]
No. The app disappears and it is in background. Don´t crash. Someone idea?

**Comment by coomar2841** [comment]
Can you please post logs from your device?

**Comment by AlexPGA** [comment]
Dont have log. 
It is a strange case . The app does not fail or experiencing an error, just the app moves to the background and closes the activity as normal. The action is as if the library had a finish () ; only close the app the first time we used it . Other times it works perfectly. Thanks for your reply
